changelog/unreleased/issue-2858

```diff
@@ -0,0 +1,7 @@
+Enhancement: Support filtering snapshots by tag and path in the stats command
+
+We've added filtering snapshots by `--tag tagList` and by `--path path`
+to the`stats` command.
```

Comment:
Please change this to:

~~~
We've added support for filtering snapshots by `--tag tagList` and `--path path`
in the `stats` command.
~~~

Reply:
I would add in the changelog that this change now allows filtering for all snapshots and not only for "latest" snapshot.

Maybe also add here a reference to the forum:
https://forum.restic.net/t/stats-for-a-host-and-filtered-snapshots/3020